src/__tests__/__snapshots__/RootReducer.test.js.snap

```diff
@@ -81,6 +81,7 @@ Object {
     "plugins": Array [],
   },
   "exchangeRates": Object {},
+  "isLoggedOut": true,
```

Comment:
Please rename this to `loginSceneStarting`, or something like that. This has more to do with component state than core account state, so the name is confusing.